lib/Raven/Processor/SanitizeDataProcessor.php

```diff
@@ -98,7 +98,11 @@ public function sanitizeHttp(&$data)
             }
         }
         if (!empty($http['data']) && is_array($http['data'])) {
-            array_walk_recursive($http['data'], array($this, 'sanitize'));
```

Comment:
There is no need for another function call here (lambda). Please use the callable notation with array syntax